legend-pure-ide-light/src/main/java/org/finos/legend/pure/ide/light/session/PureSession.java

```diff
@@ -48,17 +44,26 @@
 import java.io.InputStreamReader;
 import java.io.OutputStream;
 import java.nio.file.Paths;
-import java.util.Optional;
+import java.util.*;
+import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.regex.Pattern;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 import static org.finos.legend.pure.ide.light.helpers.response.ExceptionTranslation.buildExceptionMessage;
 
+
 public class PureSession
 {
-    private final PureRuntime pureRuntime;
+    public static final String LEGEND_TEST_SERVER_HOST = "legend.test.server.host";
+    public static final String LEGEND_TEST_SERVER_PORT = "legend.test.server.port";
+    public static final String LEGEND_TEST_H2_PORT = "legend.test.h2.port";
```

Comment:
I think certain choices wont work by updating this property dynamically.  Will try to share our problems